intent-classification-v0/ml_modules/bert_classifier.py

```diff
@@ -0,0 +1,71 @@
+from dotenv import load_dotenv
+from typing import Dict, Tuple, Set, Optional, List, Union
+import logging
+import os
+import requests
+
+load_dotenv(".env")
+
+# Add logging.
+logging.basicConfig(level=logging.INFO)
+
+API_URL = "https://api-inference.huggingface.co/models/deprem-ml/bert-fine-tuned-intent"
```

Comment:
This model is outdated. "multilabel_earthquake_tweet_intent_bert_base_turkish_cased" is the updated one.